pdb2sql/StructureSimilarity.py

```diff
@@ -498,18 +472,19 @@ def compute_lrmsd_pdb2sql(self,exportpath=None,method='svd'):
         Returns:
             float: L-RMSD value of the conformation
         """
+        backbone = ['CA', 'C', 'N', 'O']
+
         # create the sql
         sql_decoy = pdb2sql(self.decoy,sqlfile='decoy.db')
         sql_ref = pdb2sql(self.ref,sqlfile='ref.db')
 
         # extract the pos of chains A
-        xyz_decoy_A = np.array(sql_decoy.get('x,y,z',chainID='A'))
-        xyz_ref_A = np.array(sql_ref.get('x,y,z',chainID='A'))
+        xyz_decoy_A = np.array(sql_decoy.get('x,y,z',chainID='A', name=backbone))
```

Comment:
why only backbone ?

Reply:
Only backbone is needed in L-RMSD calculation, for both fitting and rmsd calculation. Here is the original reference https://doi.org/10.1002/prot.10393.